app/code/Dev/ProductComments/Block/Widget/Samplewidget.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Dev\ProductComments\Block\Widget;
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\View\Element\Template;
+use Magento\Widget\Block\BlockInterface;
+
+class Samplewidget extends Template implements BlockInterface
```

Comment:
Magento follows camelCase naming pattern. In your case, it should be SampleWidget.

Also, I think SampleWidget is bad naming because its file name should be related to module and its functionality. I think that better would be to name it CommentsWidget.php

Reply:
if Its camelCase it shouldn't be sampleWidget ? commentsWidget